src/lib.rs

```diff
@@ -467,21 +467,31 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
 
                     let mut ui = ui.child_ui(rect, Default::default());
                     ui.push_id(node_index, |ui| {
-                        ScrollArea::both()
-                            .id_source(
-                                self.id
-                                    .with((tab_viewer.title(tab).text(), "egui_dock::Tab")),
-                            )
-                            .show(ui, |ui| {
-                                Frame::none().inner_margin(tab_viewer.inner_margin()).show(
-                                    ui,
-                                    |ui| {
-                                        let available_rect = ui.available_rect_before_wrap();
-                                        ui.expand_to_include_rect(available_rect);
-                                        tab_viewer.ui(ui, tab);
-                                    },
-                                );
-                            });
+                        if style.tab_include_scrollarea {
+                            ScrollArea::both()
+                                .id_source(
+                                    self.id
+                                        .with((tab_viewer.title(tab).text(), "egui_dock::Tab")),
+                                )
+                                .show(ui, |ui| {
+                                    Frame::none().inner_margin(tab_viewer.inner_margin()).show(
+                                        ui,
+                                        |ui| {
+                                            let available_rect = ui.available_rect_before_wrap();
+                                            ui.expand_to_include_rect(available_rect);
+                                            tab_viewer.ui(ui, tab);
+                                        },
+                                    );
+                                });
+                        } else {
+                            Frame::none()
+                                .inner_margin(tab_viewer.inner_margin())
+                                .show(ui, |ui| {
+                                    let available_rect = ui.available_rect_before_wrap();
+                                    ui.expand_to_include_rect(available_rect);
```

Comment:
Lines 490 and 491 can be safely deleted, their purpose was only to allocate space for the scroll area.